build/azure-pipelines.release-template.yml

```
@@ -202,10 +202,10 @@ stages:
         strategy:
           matrix:
             # Skip windows smoke tests until we can fix the windows agent
-            #windows:
-            #  poolName: "windows"
-            #  vmImage: ""
-            #  GOMODCACHE: "C:/Users/porterci/go/pkg/mod"
+            windows:
+              poolName: "windows"
+              vmImage: ""
+              GOMODCACHE: "C:/Users/porterci/go/pkg/mod"
```

Are these slashes the right way? 
```suggestion
              GOMODCACHE: "C:\Users\porterci\go\pkg\mod"
```

---

good question, probably not? Let me see what happens